docs/how-to/code-signing/android.rst

```diff
@@ -77,8 +77,13 @@ used as an upload key for the Google Play store, we call the key "upload-key".
 
 We use a password of ``android``. This is the `default password for common
 Android keystores <https://developers.google.com/android/guides/client-auth>`__.
-You can change the password if you like. It is more important to limit who
-has access to the keystore file than to change the password.
+You should change the password. It is most critical to limit who
+has access to the keystore file, but changing the password is also important.
+
+.. admonition:: Android Keystore Password
+
+  Previously, it has been considered safe to use the default password. However,
+  recent advancements in SHA-1 attacks make that inadvisable.
```

Comment:
If an attacker has the file and it uses the default password, then they can already access the key; they don't need to break any hashes. If an attacker doesn't have the file, then surely the password doesn't matter. So what's the significance of SHA-1 here?

Reply:
If the attacker doesn't have the file, the password enables them to get the SHA-1 fingerprint of the file. As of 2020 in a research environment SHA-1 fingerprint collisions could be generated accurately for $11k, and since then the decreases in relevant hardware prices make it even cheaper. 

I'm hoping to hint which direction somebody should go in to learn more without trying to make *everybody* catch up on it. If you've got a better idea for a hint (or if this project prefers not to leave hints), I'm very open to suggestions! This is my first contribution for this project 😅

Reply:
Noting here that we chatted about this in person and then I did some extra research, and we're pretty sure this attack still requires access to the keystore file. It's still important to update the password, but maybe not bad enough to warrant the admonition, since (in its current form) it's likely to confuse people who aren't super into this specific topic. 

Accordingly, I'll remove the admonition but keep the revised wording earlier on.